bundles/binding/org.openhab.binding.lightwaverf/src/main/java/org/openhab/binding/lightwaverf/internal/LightwaveRfHeatPoller.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lightwaverf.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.openhab.binding.lightwaverf.internal.command.LightwaveRFCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+ /**
+  * @author Neil Renaud
+  * @since 1.7.0
+ */
+public class LightwaveRfHeatPoller  {
```

Comment:
yet another class which shares many code with xxxSender and xxxReceiver (Superclass?)